src/main/com/aca/homework/week9/invoice/service/InvoiceService.java

```diff
@@ -0,0 +1,44 @@
+package com.aca.homework.week9.invoice.service;
+
+import java.io.FileNotFoundException;
+import java.util.Arrays;
+
+public class InvoiceService {
+    private Invoice[] invoices;
+
+    public void load(String filename) throws FileNotFoundException {
+        invoices = new InvoiceReader().read(filename);
```

Comment:
please do not instantiate the InvoiceReader here; instead, require it in the constructor. by doing this you will not be able to extend from InvoiceReader and write unit tests without a real text file